structures/User.ts

```diff
@@ -0,0 +1,21 @@
+import Base from './Base';
+
+export enum UserPremiumType {
+  NONE,
+  NITRO_CLASSIC,
+  NITRO,
+}
+
+export default interface User extends Base {
+  avatar?: string;
+  bot?: boolean;
+  discriminator?: string;
+  email?: string;
+  flags?: number;
+  locale?: string;
+  mfa_enabled?: boolean;
+  premium_type?: UserPremiumType;
```

Comment:
I thought this one oauth only, is it not?

Reply:
Correct